### DISCUSSION TITLE
working in REST API for Chart

### DISCUSSION THREAD

**Discussion by soniveer1992** [sinaptik-ai/pandas-ai#689] [General]
Hi All , 

I am new in python , I am creating REST API ,  and trying to create chart .

its creating successfully chart and working fine (VS Code), But i want to get image URL or any supporting format that i can use in UI application (SAP UI5).

Kindly help

**Comment by gventuri** [comment]
@soniveer1992 what we recommend is to getting the path where the chart is saved (as an image) and serve it (or server the base64 version of that).

**Reply by soniveer1992** [reply]
thanks for your reply , is it possible to save image in base64 format ?? in response so that i can bind image to UI
